templates/template-scala-parallel-recommendation/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Is the license file necessary for each template after they are incorporated into this repository?